openai_connector/models/openai_vision_session.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import json
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class OpenaiVisionSession(models.Model):
+    _name = "openai.vision.session"
+    _inherit = ["api.call.mixin"]
+    _description = "OpenAI Vision Session"
+
+    name = fields.Char(required=True)
+    model = fields.Selection(
```

Comment:
To be a bit more specific.
```suggestion
    llm_model = fields.Selection(
```

Reply:
Done